pkg/cmd/server/start/start_master.go

```diff
@@ -478,6 +476,25 @@ func (m *Master) Start() error {
 			// continuously run the scheduler while we have the primary lease
 			go runEmbeddedScheduler(m.config.MasterClients.OpenShiftLoopbackKubeConfig, m.config.KubernetesMasterConfig.SchedulerConfigFile, m.config.KubernetesMasterConfig.SchedulerArguments)
 
+			// OpenShift uses a different default volume recycler template than
```

Comment:
You need to remove `newPersistentVolumeRecyclerPodTemplate` from  `pkg/cmd/server/start/start_kube_controller_manager.go`, right?

Reply:
removed